src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: "
+                + " the first leg is: "
+                + firstLeg
+                + " and the second leg is: "
+                + secondLeg
+                + ", the color is: "
+                + color);
+    }
```

Comment:
Use the 'name()' method instead of 'toString()' when you want to get the String representation of enum constants. Ensure that the 'color' field in the super class 'Figure' uses 'name()' if it's an enum type.